src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -200,28 +156,19 @@ public void testEnumsAndAnnotations() throws Exception {
             "29: " + getCheckMessage(MSG_WRITE_TAG, "@incomplete",
                 "This annotation field needs more code... // violation"),
         };
-        verify(checkConfig, getPath("InputWriteTagEnumsAndAnnotations.java"), expected);
+        verifyWithInlineConfigParser(getPath("InputWriteTagEnumsAndAnnotations.java"), expected);
```

Comment:
good step forward.
now config is loaded from Input file - https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTagEnumsAndAnnotations.java

please extend trailing comments `//violation` with actual message, see example at https://github.com/checkstyle/checkstyle/blob/a8f065738de6c4ff917a2ee57521f6cf8718f2ba/src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleCompactNoArray.java#L13